extensions/postgres/schema/xdb_sys_schema.sql

```diff
@@ -0,0 +1,61 @@
+CREATE TABLE xdb_sys_current_process_executions(
+    namespace VARCHAR(15) NOT NULL,
+    process_id VARCHAR(255) NOT NULL,
+    process_execution_id uuid NOT NULL,
+    PRIMARY KEY (namespace, process_id)
+);
+
+CREATE TABLE xdb_sys_process_executions(
+    namespace VARCHAR(15) NOT NULL, -- for quick debugging
+    id uuid NOT NULL,
+    process_id VARCHAR(255) NOT NULL,
+    --
+    is_current BOOLEAN NOT NULL , -- for quick debugging
+    status SMALLINT, -- 0:undefined/1:running/2:completed/3:failed/4:timeout/5:terminated
```

Comment:
enum?

Reply:
yea technically we could use Postgres enum. 
However, it is difficult to write the code with enum if the row struct doesn't have a string type for this status.
If the row struct uses an enum, it will make it difficult for other database like SQLServer/Oracle to write code with it. 

This is already the case when I use UUID in Postgres for ProcessExecutionId. In MySQL, there is no UUID type(it has string with UUID helpers). So I have to make a ProcessExecutionIdString in addition to ProcessExecutionId in the struct. But UUID is quite important because otherwise using BYTEA of Postgres will make the debugging/query in psql much harder. So that additional field is worthy.

So to make it easier for all the database types, I would suggest not to use enum here.